recipe/meta.yaml

```diff
@@ -63,7 +64,8 @@ requirements:
     - libpng >=1.6.23,<1.7
     - openblas 0.2.19|0.2.19.*     # [not win]
     - blas 1.1 {{ blas_variant }}  # [not win]
-    - ffmpeg                       # [not win]
+    - ffmpeg >=3.2.3,<3.2.6        # [not win]
```

Comment:
Mirrors the pinning change coming in https://github.com/conda-forge/conda-forge.github.io/pull/397, for primary reason pointed out here: https://github.com/conda-forge/opencv-feedstock/issues/82#issuecomment-309084160. In short, ffmpeg 3.3 has only 82.57% backward compatibility (https://abi-laboratory.pro/tracker/timeline/ffmpeg/).